entity-types/ext-dbt_failed_test_row/definition.yml

```diff
@@ -0,0 +1,47 @@
+domain: EXT
+type: DBT_FAILED_TEST_ROW 
+
+synthesis:
+  rules:
+    - identifier: entity_id 
+      name: entity_name 
+      encodeIdentifierInGUID: true
+      conditions:
+        - attribute: eventType
+          value: dbt_failed_test_row 
+        - attribute: entity_name
+          present: True
+        - attribute: dbt_source
+          present: True
+
+  tags:
+    team:
+    project_name:
+    environment_name:
+    slack_mentions:
+    field_1:
+    field_2:
+    field_3:
+    field_4:
+    field_5:
+    field_6:
+    field_7:
+    field_8:
+    field_9:
+    field_10:
```

Comment:
not my domain, but why these don't have an actual name of what they are?

Reply:
These represent database queries where some data is tested. Each test can query different tables and columns. Therefore, we generalized this to fields 1 - 10.